index.html

```diff
@@ -141,620 +141,809 @@
 </head>
 
 <body>
-<section id='abstract'>
-    <p> A <em>verifiable claim</em> is a qualification, achievement, quality, or piece of
-    information about an <a>entity's</a> background such as a name, government ID, payment
-    provider, home address, or university degree. Such a claim describes a quality or qualities,
-    property or properties of an <a>entity</a> which establish its existence and uniqueness. The use cases
-    outlined here are provided in order to make progress toward possible future standardization and
-    interoperability of both low and high-stakes <a>claims</a> with the goals of storing,
-    transmitting, and receiving digitally verifiable proof of attributes such as qualifications and
-    achievements. The use cases in this document focus on concrete scenarios that the technology
-    defined by the group should address. </p>
-</section>
+  <section id='abstract'>
+    <p> 
+A <em>verifiable claim</em> is a qualification, achievement, quality, or piece of
+information about an <a>entity's</a> background such as a name, government ID, payment
+provider, home address, or university degree. Such a claim describes a quality or
+qualities, property or properties of an <a>entity</a> which establish its existence and
+uniqueness. The use cases outlined here are provided in order to make progress toward
+possible future standardization and interoperability of both low and high-stakes
+<a>claims</a> with the goals of storing, transmitting, and receiving digitally verifiable
+proof of attributes such as qualifications and achievements. The use cases in this
+document focus on concrete scenarios that the technology defined by the group should
+address. 
+   </p>
+  </section>
 
-<section id='sotd'>
-    <p>This document represents a concise but limited collection of use cases readers should review
-    in conjunction with the proposed Charter for a Verifiable Claims Working Group.</p>
-    <p>Comments regarding this document are welcome. Please
-      or file
-      directly on <a href="https://github.com/w3c/vc-use-cases/issues/">GitHub</a>,
-      or send them to
-      <a href="mailto:public-vc-comments@w3.org">public-vc-comments@w3.org</a> 
-      (<a href="mailto:public-vc-comments-request@w3.org?subject=subscribe">subscribe</a>,
-      <a href="https://lists.w3.org/Archives/Public/public-vc-comments/">archives</a>).</p>
-</section>
+  <section id='sotd'>
+    <p>
+This document represents a concise but limited collection of use cases readers should
+review in conjunction with the proposed Charter for a Verifiable Claims Working Group.
```

Comment:
I'm not sure this sentence still accurately describes the purpose of the use cases document, since the charter is no longer proposed.

Reply:
Updated